app/code/community/Bolt/Boltpay/Helper/UrlTrait.php

```diff
@@ -84,9 +115,14 @@ public function getApiUrl($storeId = null)
      */
     public function getJsUrl($storeId = null)
     {
-        return  Mage::getStoreConfigFlag('payment/boltpay/test', $storeId) ?
-            $this->jsUrlTest :
-            $this->jsUrlProd ;
+        if (Mage::getStoreConfig('payment/boltpay/test', $storeId)) {
+            return $this->getCustomURLValueOrDefault(
+                Mage::getStoreConfig('payment/boltpay/custom_js', $storeID),
+                $this->jsUrlTest
+            );
+        } else {
```

Comment:
@vitaliyreznikov  Can we use the ternary operator here? If you don't want to use it, can you remove the "else" condition and simply return $this->apiUrlProd?